sql-plugin/src/main/scala/com/nvidia/spark/rapids/TypeChecks.scala

```diff
@@ -1265,6 +1265,26 @@ object SupportedOpsDocs {
     println("decimals are disabled by default in the plugin, because it is supported by a small")
     println("number of operations presently, which can result in a lot of data movement to and")
     println("from the GPU, slowing down processing in some cases.")
+    println("Result `Decimal` precision and scale follow the same rule as CPU mode in Apache Spark:")
```

Comment:
This looks fine. The only issue now is that you need to do a build so that the generated file is updated and check that file in too.  The precommit build checks to see if you didn't do this so we can make sure that the docs are always up to date.

Reply:
Sure I just build and them pushed the changes on doc as well.

Reply:
@revans2  @sameerz 
I accidentally polluted my branch. Can I cancel this PR and open a new PR?